app/models/form/sales/questions/buyer2_relationship_to_buyer1.rb

```diff
@@ -0,0 +1,29 @@
+class Form::Sales::Questions::Buyer2RelationshipToBuyer1 < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "relat2"
+    @check_answer_label = "Buyer 2's relationship to buyer 1"
+    @header = "What is buyer 2's relationship to buyer 1?"
+    @type = "radio"
+    @hint_text = ""
+    @page = page
+    @answer_options = ANSWER_OPTIONS
+    @conditional_for = {
+      "otherrelat2" => %w[X],
+    }
+    @hidden_in_check_answers = {
+      "depends_on" => [
+        {
+          "relat2" => "X",
```

Comment:
i.e. This is hidden in check answers when the answer is other? Why?

Reply:
When it's Other we actually just the text the user input rather than the word Other as well - I think that's what we want? Same format as here: https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/909/files#diff-8ea21e0b734464835ea4577b4152b914cbabde55e6a35c36f02edfb69f6ce242

Reply:
Right that makes sense, sorry early morning :sweat_smile: